plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -183,7 +183,8 @@ class GeneralServiceFormSection extends Component {
           key={index}
           onRemove={this.props.onRemoveItem.bind(this, {
             value: index,
-            path: "containers"
+            path: "containers",
+            noPropagate: true
```

Comment:
I want the `onRemoveItem()` implementation to stop event propagation when invoked from this context (because otherwise it will trigger the container click event too). There might be a better way of doing this though.